client/src/pages/Followers.jsx

```diff
@@ -1,29 +1,23 @@
-import React from 'react';
+import React, {useState} from 'react';
 import Follower from '../components/Follower';
 
-export default class Followers extends React.Component {
+export default function Followers(){
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      isFirstLoad: true,
-      isLoading: false,
-      isNewUser: false,
-      followers: [],
-      newFollowers: [],
-      unFollowers: [],
-      userId: '',
-      message: ''
-    }
-  }
+  const [isFirstLoad, setIsFirstLoad] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isNewUser, setIsNewUser] = useState(false);
+  // const [followers, setFollowers] = useState([]);
+  const [newFollowers, setNewFollowers] = useState([]);
+  const [unFollowers, setUnFollowers] = useState([]);
+  const [message, setMessage] = useState('');
+  const [userId, setUserId] = useState('');
 
-  handleUserIdChange = (event) => {
-    this.setState({ userId: event.target.value })
+ function handleUserIdChange(event){
+    setUserId(event.target.value);
   }
```

Comment:
Maybe we do not need a function here anymore. I saw some examples about the sample usage as, setUserId(event.target.value) in render part. Kaan hocam will lighten us up.

Reply:
I think both are fine but your suggestion could be better for the sake of simplicity 👍